app/views/content/shared/block-promos/_adviser_findsalary.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="row">
```

Comment:
We are not using this anywhere - presume `/funding-and-support/salaried-teacher-training` needs adding to this PR?